indico/modules/events/contributions/templates/display/_paper_reviewing.html

```diff
@@ -0,0 +1,112 @@
+{% from 'forms/_form.html' import form_header, form_rows, form_footer %}
+
+{% set UPLOAD_NEEDED_STATUSES = ('To be corrected', none) %}
```

Comment:
oh my, who thought back then it'd be a good idea to use strings like this as the status identifiers